website_sale_hide_price/models/product_template.py

```diff
@@ -38,3 +38,21 @@ def _get_combination_info(
             }
         )
         return combination_info
+
+    def _search_render_results(self, fetch_fields, mapping, icon, limit):
+        """Hide price on the search bar results""""
```

Comment:
Too many quote marks :sweat_smile: 
```suggestion
        """Hide price on the search bar results"""
```